gen/nvml/return.go

```diff
@@ -19,6 +19,10 @@ func ErrorString(Result Return) string {
 	return nvmlErrorString(Result)
 }
 
+func (r Return) Value() Return {
```

Comment:
Just to be clear, this isn't used at the moment?

Reply:
It's not used anywhere in this repo, correct.
But I am using it in a wrapper I am building.